pysimplestplus/utils/values.py

```diff
@@ -241,6 +241,14 @@ def to_choice(self):
 
     def added_to(self, other):
         if isinstance(other, Numerical):
+            result = self.value + other.value
+            if not -999999999.999999 <= result <= 999999999.999999:
+                return None, SemanticError(
+                    self.pos_start,
+                    other.pos_end,
+                    f"{result} is outside the range of 999999999.999999 to -999999999.999999",
+                    self.context,
+                )
```

Comment:
Goods to, pero use the max of num and deci sa constants.py instead of hardcoding

tapos, you can create a method nalang sa Numerical class na check_value_range(self, value) tapod dyan mo itransfer tong code na to tapos magrereturn ng None or SemanticError, balr sa bawat operation, gamitin mo nalang tong method na to before ireturn yung result, kapag hindi None yung error, return mo yung tuple na (None, error)

Reply:
Done all revisions.